.github/workflows/check-standard.yaml

```diff
@@ -1,15 +1,16 @@
 # For help debugging build failures open an issue on the RStudio community with the 'github-actions' tag.
 # https://community.rstudio.com/new-topic?category=Package%20development&tags=github-actions
+name: R-CMD-check
+
 on:
   push:
     branches:
       - main
+      - issue170_fix
```

Comment:
Let's remove this line before merging. The main branch would still run this workflow.

Reply:
> Just want clarification. Are you referring to the line with the name of my branch?